src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -604,13 +599,13 @@ internal static void FindSerializationInfo(Type type)
                                     typeInfo.Name,
                                     assembly.GetName().Name);
 
-                            var register = typeInfo.GetMethod("Register", Type.EmptyTypes);
+                            var register = typeInfo.GetMethod("Register", new[] {typeof(SerializationManager)});
                             if (register != null)
```

Comment:
Since you made this change to the expected signature (breaking change, please make a note in the changelog!), you need to also update the Warning message you output when `register` is null.